vnet.tf

```diff
@@ -1,18 +1,41 @@
+# # Generic locals
+# locals {
+#   common_tags = module.ctags.common_tags
+# }
+
+# module "ctags" {
+#   source      = "git::https://github.com/hmcts/terraform-module-common-tags.git?ref=master"
+#   environment = var.env
+#   product     = var.product
+#   builtFrom   = var.builtFrom
+# }
```

Comment:
not needed as you are on Jenkins

it will inject a variable `common_tags` for you to use instead

Reply:
modified